pod/perlre.pod

```diff
@@ -1350,25 +1350,36 @@ X</p> X<p modifier>
 
 =head2 Quoting metacharacters
 
-Backslashed metacharacters in Perl are alphanumeric, such as C<\b>,
-C<\w>, C<\n>.  Unlike some other regular expression languages, there
-are no backslashed symbols that aren't alphanumeric.  So anything
-that looks like C<\\>, C<\(>, C<\)>, C<\[>, C<\]>, C<\{>, or C<\}> is
-always
-interpreted as a literal character, not a metacharacter.  This was
-once used in a common idiom to disable or quote the special meanings
-of regular expression metacharacters in a string that you want to
-use for a pattern. Simply quote all non-"word" characters:
+To cause a metacharacter to match its literal self, you quote (or
+escape) it by preceding it with a backslash.  Unlike some other
+regular expression languages, any sequence consisting of a backslash
+followed by a non-alphanumeric matches that non-alphanumeric, literally.
+So things like C<\\>, C<\(>, C<\)>, C<\[>, C<\]>, C<\{>, or C<\}> are
+always interpreted as the literal character that follows the backslash.
+
+(That's not true when an alphanumeric character is preceded by a
+backslash.  There are a few such sequences, like C<\w>, which have
+special meaning for Perl.)
+
+But with a non-alphanumeric, it will match literally.  Hence simply
+quoting all non-"word" characters can be used to disable the special
+meanings of regular expression metacharacters in a string that you want
+to use for a pattern. 
 
     $pattern =~ s/(\W)/\\$1/g;
 
-(If C<use locale> is set, then this depends on the current locale.)
-Today it is more common to use the C<L<quotemeta()|perlfunc/quotemeta>>
+(If C<use locale> is in effect, then this depends on the current locale.)
+
+This template used to be a common paradigm, but these days it is more
```

Comment:
It's not really the purpose of this PR, but I think it's a little misleading to say that quotemeta "disables metacharacters special meanings" without also saying that it actually does a very similar thing to the example just above, without being locale dependent. i.e. it's exactly equivalent to `s/([^A-Za-z_0-9])/\\$1/g` or `s/(\W)/\\$1/ag`.

Reply:
With the additional very useful feature that it can be interpolated with `\Q` and `\E` rather than applied to the whole pattern.

Reply:
@Grinnz please check if new wording is ok